README.md

```diff
@@ -12,24 +12,27 @@ Bitcoin stock price for your RPi Zero
     ```
     Interfacing Options -> SPI
    ```
-2. Install eInk display drivers
+2. Installing dependencies via apt
     ```
-    git clone https://github.com/waveshare/e-Paper.git
-    cd e-Paper-master/RaspberryPi_JetsonNano/python/
-    pip3 install -e $(pwd)
+    sudo apt update
+    sudo apt install -y libtiff5 libopenjp2-7 libopenjp2-7 libatlas-base-dev
     ```
-    for more information refer to: https://www.waveshare.com/wiki/2.13inch_e-Paper_HAT
-3. Download Zero BTC Screen
+3. Install eInk display drivers
+    ```
+    pip3 install image numpy RPi.GPIO spidev Pillow
+    git clone https://github.com/waveshare/e-Paper.git ~/e-Paper
+    pip3 install ~/e-Paper/RaspberryPi_JetsonNano/python/
     ```
-    cd ~
-    git clone https://github.com/dr-mod/zero-btc-screen.git
+    for more information refer to: https://www.waveshare.com/wiki/2.13inch_e-Paper_HAT
+4. Download Zero BTC Screen
     ```
-4. Run it 
+    git clone https://github.com/dr-mod/zero-btc-screen.git ~/zero-btc-screen
     ```
-    cd zero-btc-screen
-    python3 main.py
+5. Run it 
+    ```Pillow
```

Comment:
Mistype?

Reply:
Yes, I have no idea on how it got in there!